pages/checkout.tsx

```diff
@@ -1,5 +1,11 @@
 import { CheckoutButton } from "@components/CheckoutButton";
 
+// test price id corresponding to a product in stripe
+const testPriceId = "price_1J1GuzL97YpjuvTOePyVbsRh";
```

Comment:
Will this be removed in the future?

Reply:
this is just a test price ID but when we actually implement the page then we'll hardcode it with the correct price ID - does that make sense?